### DISCUSSION TITLE
Confused about the masking. How can I improve

### DISCUSSION THREAD

**Discussion by TwistNemo** [AstroBridge/BDLensing#97] [Q&A] [answered]
Here is my code link: 

https://drive.google.com/file/d/1Yr6myqaVjW39UifvLL0vXtKFsgCglr8Z/view?usp=sharing

**Comment by Rajit13** [comment]
Upload the code to your local forked repository. It would be easier to follow.

From what I noticed
1. Your mask is too big considering the background object so it is maybe covering some of the lens arc.
2. There is more masking to be done near 4'o clock.
3. You may need to apply double sersic to reduce the blue residuals.
4. You may need to cover the central source.

**Reply by TwistNemo** [reply]
Thanks a lot.

**Comment by rafee-adnan** [comment, marked as answer]
I think the following suggestions will improve your model:

1. As you can see from the decomposition plot(labeled 'all components convolved'), it has created a light source right where you put the mask(4'o clock). I think you should run your code without masking the region.
2. Your lens model's gamma value is quite low, try setting the lower limit higher, around 1.5. 
3. And I agree with adding another sersic profile to your lens galaxy's light model.
4. Make the image of the reconstructed source visible if it is too tiny to see. For that, you can change `deltaPix_source` inside the `model_plot.source_plot()` Best of luck!

<img width="348" alt="Screenshot 2023-06-01 at 5 52 55 PM" src="https://github.com/ajshajib/BDLensing/assets/76589056/b6816602-2979-4ee1-a2af-b6f849ae3807">
<img width="355" alt="Screenshot 2023-06-01 at 6 19 11 PM" src="https://github.com/ajshajib/BDLensing/assets/76589056/ca57173e-addc-4161-aa14-b2e44e0996d8">

**Reply by TwistNemo** [reply]
Thanks a lot, Vai
